projects/canopy/src/lib/forms/checkbox-group/checkbox-group.component.html

```diff
@@ -1,4 +1,8 @@
-<fieldset [attr.aria-describedby]="ariaDescribedBy">
+<fieldset
+  [attr.aria-describedby]="ariaDescribedBy"
+  [attr.tabindex]="focus ? '-1' : null"
```

Comment:
I considered adding the `tabindex` to the `lgFocus` directive but decided not to. The reason is that sometime you could be targeting a link, a button or an input and you wouldn't want to change the `tabindex` to `-1`. Unless we check the tags of every focusable element and then either add it or not but I thought this was better.

Reply:
Would you mind just explaining the negative tabIndex on the fieldset? So if it has 'focus' then we remove it from the tabIndex. Is this to give it focus but not steal tab control?

Reply:
You don't really want the fieldset to be reachable via keyboard navigation, but you want it to be focusable with Javascript, that's why I used that.

Reply:
> I considered adding the tabindex to the lgFocus directive but decided not to.

An alternative approach could be to add a new Input property to the Focus directive, something like: 

`@Input() lgSetTabIndexOnFocus: boolean`

which could apply the `-1` if `true`, or remove it if not, from within the directive class. I think it would work as we already get access the host element in the directive.

E.g.

```
<fieldset
  [lgFocus]="lgFocus"
  [lgSetTabIndexOnFocus]="true"
```

If it works, you could avoid needing to implement this `tabindex` change on each individual input component.